src/pages/index.js

```diff
@@ -9,12 +9,14 @@ export default function Home() {
   const { livestreamUrl } = useSiteMetadata()
   
   return (
-    <Layout>
-      <Player
-        streamUrl={livestreamUrl}
-        preloadType="auto"
-      />
-      <Calendar />
+    <Layout className={isPlayerLoaded && isCalendarLoaded ? "ready" : ""}>
+      <div className="content">
```

Comment:
Why is this wrapper necessary? If it is, I believe is should also be embedded in Layout.js